lib/absinthe/phase/subscription/result.ex

```diff
@@ -5,10 +5,54 @@ defmodule Absinthe.Phase.Subscription.Result do
   # subscription
 
   alias Absinthe.Blueprint
+  alias Absinthe.Blueprint.Continuation
+  alias Absinthe.Phase
 
   @spec run(any, Keyword.t()) :: {:ok, Blueprint.t()}
-  def run(blueprint, topic: topic) do
+  def run(blueprint, options) do
```

Comment:
We should probably want to at least guarantee a `topic` here and pattern match as such, right?

Reply:
Yeah, that's probably a reasonable point - the change loses an existing check. I'll tweak that now.